CHANGELOG.md

```diff
@@ -1,11 +1,76 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
 
-## Release 0.1.0
+## [v1.0.0](https://github.com/voxpupuli/puppet-nftables/tree/v1.0.0) (2020-12-14)
 
-**Features**
+[Full Changelog](https://github.com/voxpupuli/puppet-nftables/compare/0ba57c66a35ed4e9b570d8a6315a33a1c4ba3181...v1.0.0)
 
-**Bugfixes**
+**Breaking changes:**
 
-**Known Issues**
+- switch not the server naming [\#42](https://github.com/voxpupuli/puppet-nftables/pull/42) ([duritong](https://github.com/duritong))
```

Comment:
oh my typo made it into the release notes, maybe fix that?

Reply:
oh and why is this a known issue?

Reply:
It's better to fix the resolved MR and regenerate CHANGELOG as it is regenerated from scratch every time.

Looking up the other question.

Reply:
Added the enhancement tag also which I guess will remove it to a normal category.